source/MRMesh/MRToolPath.cpp

```diff
@@ -362,6 +367,301 @@ ToolPathResult constantZToolPath( const Mesh& inputMesh, const ToolPathParams& p
     return res;
 }
 
+template<typename T>
+std::pair<T, T> findMinMax( const std::vector<T>& vec )
```

Comment:
looks like this function can be useful in other places too, also better name it `parallelMinMax`